config/settings.py

```diff
@@ -241,3 +241,9 @@
 EMAIL_USE_SSL = False
 EMAIL_HOST_USER = os.getenv("EMAIL_HOST_USER", "information.mevent@gmail.com")
 EMAIL_HOST_PASSWORD = os.getenv("EMAIL_HOST_PASSWORD")
+
+# Authentication
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+    'backends.authentication.AuthenticationWithoutPassword',
+)
```

Comment:
> When somebody calls [django.contrib.auth.authenticate()](https://docs.djangoproject.com/en/5.0/topics/auth/default/#django.contrib.auth.authenticate) – as described in [How to log a user in](https://docs.djangoproject.com/en/5.0/topics/auth/default/#how-to-log-a-user-in) – Django tries authenticating across all of its authentication backends. If the first authentication method fails, Django tries the second one, and so on, until all backends have been attempted.

If I understood right, it means that any authenticate call for an email verified user now will work, even if he could't pass other backends: for example he didn't write password and still is being authenticated.